Fixes NtosNet Downloader & App themes

JohnFulpWillard opened the pull request:
## About The Pull Request

NtOS' downloading app is the only application where PC device theme is actually used by the app directly, instead of handled by NtOS Window. I forgot about this, and removed it from the data, breaking the TGUI page.

Also two edits;
- NtosWindow now takes their theme directly from PC_device_theme, this means we're not relying anymore on apps to individually give the theme they want to have, and tablets will get alerts of downloading illegal programs if they are on any theme but the syndicate one (instead of not getting the message if theyre on any but the nt one).
- Removes bad uses of ``as anything`` on stored files, because it holds more than just programs (like ordnance data).

## Why It's Good For The Game

the NtOS app now works again, sorry for the issue.

Closes https://github.com/tgstation/tgstation/issues/73493
Closes https://github.com/tgstation/tgstation/issues/73507

## Changelog

:cl:
fix: NtOS program downloader now works again, and will now properly alert you of downloading illegal ROMs if you change your theme to anything that isn't Syndicate.
fix: NtOS themes are now recognized by all windows.
/:cl:

JohnFulpWillard committed:
Fixes NtosNet Downloader

NtOS' downloading app is the only application where PC device theme is actually used by the app directly, instead of handled by NtOS Window. I forgot about this, and removed it from the data, breaking the TGUI page.

SyncIt21 commented:
Fixes #73493.

Also while were here can you also fix #73494? The option to open hardware configuration was still hanging around even after you nerfed it,

JohnFulpWillard committed:
Fixes misc tablet problems

ALL tablet apps now respect pc device theme

JohnFulpWillard commented:
I commented on that PR, I have no idea how they got there. I straight up removed that app.

SyncIt21 commented:
> I commented on that PR, I have no idea how they got there. I straight up removed that app. ![image](https://user-images.githubusercontent.com/53777086/219848937-5d331741-ca9f-46e3-82be-b864502c5e3c.png) ![image](https://user-images.githubusercontent.com/53777086/219848945-1d0b1fda-ce65-4a21-bc69-879f6826ac88.png) ![image](https://user-images.githubusercontent.com/53777086/219848983-4431a7ff-97ad-48b7-8fc9-eb9a550eb8cd.png)

LOL. No worries then it's not like people use that app anyway & we have no idea how he found it. Maybe Next time

JohnFulpWillard committed:
Fixes improper use of as anything

stored_files holds all computer files, not only programs, so we can't use as anything otherwise having non-program files (like txt or ordnance files) will break the UI.
